CONTRIBUTING.md

```diff
@@ -1,57 +1,188 @@
+  example "fixes #555." Read more GitHub guidance on
+  [closing issues via commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/).
+- We encourage you to follow the
+  [50/72 format](http://stackoverflow.com/questions/2290016/git-commit-messages-50-72-formatting).
+
 
 ## Pull requests
-- Anyone may informally review a pull request and make comments or suggestions. 
-- When a pull request is ready for review, label it `plz-review`. 
-- Include a summary of all work and changes.
-- Use `cc @username` to notify a specific team member to review a pull request.
-
-## 18F team processes
-* Use PEP8 as the coding standard for Python, but don't worry about fitting lines within 79 characters.
-* Create pull requests for all commits, even typo fixes.
-* Don't merge your own pull request. Ask a colleague to review your code and merge.
-* Pull requests contain some tests. Ideally they would contain decent test coverage.
-
-- Usually start with a verb. Like “added” or “changed” or “removed”.
-- Generally follow [50/72 format](http://stackoverflow.com/questions/2290016/git-commit-messages-50-72-formatting).
-- Reference a ticket number, where available, with "#555" if additional work is needed on the issue or “fixes #555” or any of the other [keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) if the commit resolves the issue.
-- Be sure to talk about the nature of the change you're making instead of describing the bug or task. Explain why the change was needed if it is relevant.
-
-## 18F team workflow
-
-1. Pull an issue from the current sprint only, please (unless there is an *URGENT* issue). Take an unassigned issue or re-assign an issue with the label "[stealable](https://github.com/18F/openFEC/labels/stealable)." If a non *URGENT* issue comes up, but you want to prioritize it, coordinate with the PM or tech lead on slack.
-2. Before starting, [write the dev issue checklist](https://github.com/18f/openFEC/blob/develop/CONTRIBUTING.md#development-issues-should-include).
-3. While working, submit `[WIP]` [pull requests](https://github.com/18f/openFEC/blob/develop/CONTRIBUTING.md#pull-requests) liberally.
-4. Time boxing! If you have spent half the time of the estimated size of the issue and you're not sure that you're half-way finished, notify the tech lead or PM in slack. This isn't bad, we just like knowing what's up. :)
-5. Make sure your PR includes significant test coverage.
-6. Once you're done with your PR, notify the team in slack that your PR is ready for review. Do a review of someone else's PR. (Follow the above PR guidelines.)
+- Create pull requests for all commits, even typo fixes.
+- Use [PEP 8](https://www.python.org/dev/peps/pep-0008/) as the coding
+  standard for Python, but don't worry about fitting lines within 79
+  characters.
+- Pull requests contain some tests. Ideally they would contain decent test
```

Comment:
@jenniferthibault that's a great question and something we didn't really get into in our discussions yet.  This information existed in the doc already, I just reformatted it, but it would be worth defining this further.  What are we shooting for?  At least 80% coverage, 90%, 100%?

The CodeCov integration helps in identifying areas of the code that aren't thoroughly tested and if overall test coverage drops with a change, but are we doing anything with that information now?

These are all good questions to be asking and I think it'd be good to provide a few more details around this, but we probably need to do this as a next step as opposed to answering it right now.